File: 프로그래머스/lv0/120892. 암호 해독/암호 해독.java
class Solution {
    public String solution(String cipher, int code) {
        String answer = "";
        String [] div = cipher.split("");
        for (int i = code; i <=cipher.length(); i++) {
            if (i%code==0) {
                answer += div[i-1];
            }
        }
        return answer;
    }
}

File: 프로그래머스/lv0/120892. 암호 해독/암호 해독.py
def solution(cipher, code):
    answer = ''
    #cipher -> append -> code의 배수 -> i%code == 0
    #code의 배수만 찾으면 됨 -> for문 range(code, len(cipher))
    for i in range(code, len(cipher)+1):
        if i%code == 0:
            answer += cipher[i-1]
    return answer